index.html

```diff
@@ -61,6 +61,282 @@ <h1>i'm Adan <br>
+                        </li>
+                    </ul>
+                </div>
+                <div class="see-projects">
+                    <button class="see">See Projects</button>
+                </div>
+
+            </div>
+
+
+
+            </div>
+            </div>
+        </section>
+
+        <section class="about-myself">
+            <div class="Portfolio">
+                <div class="intro">
+                    <h1>About
+                        <br>
+                        Myself</h1>
+                </div>
+                <div class="about">
+                    <p class="me">
+                        I’m a software developer! I can help you build a product , feature or website Look through some
+                        of
+                        my work and experience! If you like what you see and have a project you need coded, don’t
+                        hestiate
+                        to contact me.</p>
+                </div>
+
+                <div class="connect">
+                    <a class="btn" href="#index.html">LET'S CONNECT</a>
+                </div>
+
+                <ul class="images">
+                    <li class="column">
+
+                        <a href=""><img src="images/twitter.jpg" alt="twitter"></a>
+                    </li>
+                    <li class="column">
+
+                        <a href=""><img src="./images/github.jpg" alt="github"></a>
+                    </li>
+                    <li class="column">
+                        <a href=""><img src="./images/chil.jpg" alt="chill"></a>
+                    </li>
+                    <li class="column">
+                        <a href=""><img src="./images/M.jpg" alt="m"></a>
+                    </li>
+                    <li class="column">
+                        <a href=""> <img src="./images/i.png" alt="i">
+                            <img src="./images/n.png" alt="m"></a>
+                    </li>
+
+
+                </ul>
+                <div class="get">
+                    <button class="resume">Get my resume</button>
+                </div>
+
+                <div class="languages">
+                    <h1 class="lang">Languages</h1>
+
+                    <img class="icons" src="./images/About My Self.jpg" alt="">
+
+                    <div class="comp" id="color">
+                        <ul>
+                            <li><img class="icon" id="icon" src="./images/Frame 9.png" alt="">
+                            <!-- <li class="tec" id="icon">Javascript</li> -->
+                            <li> <img class="icon" id="icon" src="./images/Frame 10.png" alt=""></li>
+                            <!-- <li class="tec" id="icon">HTML</li> -->
+                            <li> <img class="icon" id="icon" src="./images/Frame 11.png" alt=""></li>
+                            <!-- <li class="tec" id="icon">CSS</li> -->
+
```

Comment:
- [ ] I think it would be better if you remove the commented code lines to make your codebase professional and cleaner